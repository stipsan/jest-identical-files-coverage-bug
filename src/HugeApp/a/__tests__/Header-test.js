import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../Header'

test('renders correctly', () => {
  const component = renderer.create(<Header />)
  expect(component.toJSON()).toMatchSnapshot()
})
