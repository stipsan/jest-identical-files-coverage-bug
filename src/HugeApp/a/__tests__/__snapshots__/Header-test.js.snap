// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <h1>
    Welcome!
  </h1>
</header>
`;
